fvm/evm/emulator/config.go

```diff
@@ -17,6 +17,12 @@ var (
 	bigZero = big.NewInt(0)
 )
 
+var (
+	PreviewnetPragueActivation = uint64(0)          // already on Prague for PreviewNet
+	TestnetPragueActivation    = uint64(1746039600) // Wednesday, April 30, 2025 7:00:00 PM GMT
+	MainnetPragueActivation    = uint64(1746612311) // Wed May 07 10:05:11 2025 UTC
```

Comment:
TODO(m-Peter): Update this once we set the proper time for the Pectra upgrade on Mainnet.